spec/features/editing_file_attachments/edit_file_attachment_title_spec.rb

```diff
@@ -30,4 +34,19 @@ def and_i_click_on_edit_file
   def then_i_see_the_current_attachment_title
     expect(page).to have_field("Edit title", with: @attachment_revision.title)
   end
+
+  def when_i_enter_a_new_attachment_title_and_click_save
+    @put_content_request = stub_publishing_api_put_content(@edition.content_id, {})
+
+    fill_in "file_attachment[title]", with: "Another title"
+    click_on "Save"
+  end
+
+  def then_i_see_the_new_title_on_the_attachment_index_page
+    expect(page).to have_link("Another title", href: preview_file_attachment_path(@edition.document, @attachment_revision.file_attachment))
```

Comment:
This line is quite long, this makes it a bit shorter. :) 
```ruby
expect(page).to have_link("Another title", href: preview_file_attachment_path(@edition.document,
                                                                              @attachment_revision.file_attachment))
```

Reply:
That path was so long...I've assigned it to a variable :)

Reply:
aha cool! :)